Haaretz.js

```diff
@@ -0,0 +1,43 @@
+{
+	"translatorID": "d6f64d96-aa6f-4fd3-816f-bdef842c7088",
+	"label": "Haaretz",
+	"creator": "Eran Rosenthal",
+	"target": "http://www\\.haaretz\\.co\\.il/",
```

Comment:
Talking about the English version: couldn't we make this work for haaretz.com easily, too? Looks like the structure is quite similar.